3rd-party-scripts/SimpleHTTPSServer.java

```diff
@@ -0,0 +1,105 @@
+import java.io.*;
+import java.net.InetSocketAddress;
+import java.lang.*;
+import java.net.URL;
+import com.sun.net.httpserver.HttpsServer;
+import java.security.KeyStore;
+import javax.net.ssl.KeyManagerFactory;
+import javax.net.ssl.TrustManagerFactory;
+import com.sun.net.httpserver.*;
+import javax.net.ssl.SSLEngine;
+import javax.net.ssl.SSLParameters;
+
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.net.URLConnection;
+
+import javax.net.ssl.HostnameVerifier;
+import javax.net.ssl.HttpsURLConnection;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLSession;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.X509TrustManager;
+import java.security.cert.X509Certificate;
+
+import java.net.InetAddress;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+import com.sun.net.httpserver.HttpsExchange;
+
+public class SimpleHTTPSServer {
+
+    public static class MyHandler implements HttpHandler {
+        @Override
+        public void handle(HttpExchange t) throws IOException {
+            String response = "This is the response";
+            HttpsExchange httpsExchange = (HttpsExchange) t;
+            t.getResponseHeaders().add("Access-Control-Allow-Origin", "*");
+            t.sendResponseHeaders(200, response.getBytes().length);
+            OutputStream os = t.getResponseBody();
+            os.write(response.getBytes());
+            os.close();
+        }
+    }
+
+    /**
+     * @param args
+     */
+    public static void main(String[] args) throws Exception {
```

Comment:
Why `throws Exception` if you're catching them all?

Reply:
no reason, just lazy :)